src/kernels/composable_kernel/src/kernel_wrapper/gridwise_convolution_implicit_gemm_v4r4_gen_xdlops_wrw_fp32_nchw_kcyx_nkhw_lds_double_buffer.cpp

```diff
@@ -10,7 +10,7 @@ extern "C" __global__
         FLOAT* const __restrict__ p_wei_global)
 {
 #if !(MIOPEN_USE_FP32 && CK_PARAM_PROBLEM_CONV_DIRECTION_FORWARD)
```

Comment:
CK_PARAM_PROBLEM_CONV_DIRECTION_FORWARD

change to 

CK_PARAM_PROBLEM_CONV_DIRECTION_BACKWARD_WEIGHT

Alongside, we need to change solver to set 
CK_PARAM_PROBLEM_CONV_DIRECTION_BACKWARD_WEIGHT=1
It is currently set CK_PARAM_PROBLEM_CONV_DIRECTION_FORWARD=0